### ISSUE TITLE
No video on lower display - under lane line overlay

### ISSUE THREAD

**Issue by energee** [commaai/openpilot-tools#7]
![screen shot 2019-01-24 at 5 28 03 pm](https://user-images.githubusercontent.com/842240/51712640-74e71900-1ffd-11e9-9534-0fe2d3bbea43.png)

![image](https://user-images.githubusercontent.com/842240/51712671-8fb98d80-1ffd-11e9-8c74-b8987a780bb5.png)

Using on Mac os, but @csouers saw same this issue on ubuntu.

**Comment by rbiasini** [comment]
yeah, `ui.py` fails to import `visiontest`. It will need some debug.
